applications_tests/lethe-fluid-particles/CMakeLists.txt

```diff
@@ -10,13 +10,6 @@ file(COPY particle_sedimentation_files/dem.triangulation.info DESTINATION "${CMA
 file(COPY particle_sedimentation_files/dem.triangulation_fixed.data DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/particle_sedimentation.${_build_type}/mpirun=1/")
 file(COPY particle_sedimentation_files/dem.triangulation_variable.data DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/particle_sedimentation.${_build_type}/mpirun=1/")
 
```

Comment:
I could not guess why those files were deleted and not the restart_fluidized_bed_files?

Reply:
I commited them by mistakes, you have a good eye :)